records_mover/records/pandas/prep_for_csv.py

```diff
@@ -0,0 +1,56 @@
+from pandas import DataFrame
+import pandas as pd
+from records_mover.records.schema import RecordsSchema
+from records_mover.records import DelimitedRecordsFormat
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def prep_df_for_csv_output(df: DataFrame,
+                           records_schema: RecordsSchema,
+                           records_format: DelimitedRecordsFormat) -> DataFrame:
+    # TODO: Should this take an index parameter?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker